lib/install/elm.rb

```diff
@@ -0,0 +1,21 @@
+require "webpacker/configuration"
+
+puts "Copying elm loader to #{Webpacker::Configuration.config_path}/loaders"
+copy_file "#{__dir__}/config/loaders/installers/elm.js", "config/webpack/loaders/elm.js"
```

Comment:
[Fixed](https://github.com/bkuhlmann/webpacker/commit/baf36a94bd41). NOTE: This is an issue with vue, react, and angular. Can issue a follow-up PR for those, though.